### PR TITLE
style: refactor and make css code generic

### PR DESCRIPTION
**Pull request by dev-prabhat**

### CODE REVIEW

**Review by DivyanshSareen — APPROVED** [review]
Looks good to merge

**Review by zoef-yt — APPROVED** [review]
Good to merge.

**Review by zoyatanweer — APPROVED** [review]
good to merge

**Review by zoef-yt — APPROVED** [review]

File: common.css [hunk 1]
```diff
@@ -30,6 +30,8 @@ body {
   flex-direction: row;
   justify-content: space-between;
   align-items: center;
+  background-color: var(--danger-light-color);
+  /* background-color: var(--dark-theme-shade-80); */
```
Comment (DivyanshSareen) [inline common.css]: remove comment

**Review by DivyanshSareen — APPROVED** [review]
Good to merge 👍

**Review by zoyatanweer — APPROVED** [review]
good to go

**Pull request merged**